assets/js/base/components/quantity-selector/stories/index.js

```diff
@@ -19,11 +18,13 @@ export const Default = () => {
 	const [ quantity, changeQuantity ] = useState();
 
 	return (
-		<QuantitySelector
-			disabled={ boolean( 'Disabled', false ) }
-			quantity={ quantity }
-			onChange={ changeQuantity }
-			itemName="widgets"
-		/>
+		<div style={ { width: 100 } }>
```

Comment:
This is moving the styles from [`/quantity-selector/stories/style.scss`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2818/files#diff-32004167aea03f11226ddf9e1c86df70) so they are inline. I thought it was discussed that we wanted to avoid CSS files for stories, but I might be wrong.

Reply:
Oops ya sorry, that's right, I missed that this was in a story file.